controllers/release/release_adapter_test.go

```diff
@@ -311,11 +312,22 @@ var _ = Describe("Release Adapter", Ordered, func() {
 	})
 
 	It("can return the target ReleaseLink from a given ReleaseLink", func() {
-		link, _ := adapter.getTargetReleaseLink()
+		link, _ := adapter.getActiveTargetReleaseLink()
 		Expect(reflect.TypeOf(link)).To(Equal(reflect.TypeOf(&appstudiov1alpha1.ReleaseLink{})))
 		Expect(link.Spec.Application).To(Equal("test-app"))
 		Expect(link.Spec.ReleaseStrategy).To(Equal(releaseStrategy.Name))
 		Expect(link.Namespace).Should(Equal(releaseLink.Spec.Target))
+
+		// It should return nil if the ReleaseLink has auto-release set to false
+		patch := client.MergeFrom(link.DeepCopy())
+		link.SetLabels(map[string]string{
+			"release.appstudio.openshift.io/auto-release": "false",
+		})
+		Expect(k8sClient.Patch(ctx, link, patch)).Should(Succeed())
+		Eventually(func() bool {
+			_, err := adapter.getActiveTargetReleaseLink()
```

Comment:
I would also catch the result of `_`. If the return value of the function changes in the future to not be `nil`  (for any reason) I guess keeping this would cause a wrong `success`.

Reply:
Fixed I think, let me know if not